core/lazyload/IntersectionObserverPolyfillAsset.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace luya\lazyload;
+
+use luya\web\Asset;
+
+/**
+ * The Lazyload asset files provides all required resources and implements jquery.
+ *
+ * @author Basil Suter <basil@nadar.io>
```

Comment:
@TheMaaarc if have not created this file :smile:

Reply:
Ahhh, you just don't know it yet 😄